webview-ui/src/i18n/locales/ca/common.json

```diff
@@ -6,5 +6,10 @@
 	},
 	"ui": {
 		"search_placeholder": "Cerca..."
+	},
+	"mermaid": {
+		"loading": "Generant diagrama mermaid...",
+		"render_error": "No es pot renderitzar el diagrama",
+		"copy_success": "Codi copiat"
```

Comment:
This seems like something that we probably already have in a common / shared section.

Reply:
We don't have this exact translation anywhere else, but in theory it would be nice to use a shared "Copied" or something. Doesn't seem to exist yet though.